src/PrintOrder/ComponentToPrint/ComponentToPrint.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash/get';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Col,
+  Grid,
+  KeyValue,
+  NoValue,
+  Row,
+} from '@folio/stripes/components';
+import {
+  AmountWithCurrencyField,
+  FolioFormattedTime,
+  ORDER_STATUS_LABEL,
+} from '@folio/stripes-acq-components';
+
+import {
+  DISCOUNT_TYPE,
+} from '../../components/POLine/const';
+import { LINE_FIELDS_MAP, LINE_FIELDS_LABELS } from './constants';
+import css from './ComponentToPrint.css';
+
+function PrintValue({ path, source, exportedLine }) {
+  const value = get(exportedLine, path, null) || get(source, path, null);
+  const currency = get(source, LINE_FIELDS_MAP.currency);
+
+  switch (path) {
+    case LINE_FIELDS_MAP.listUnitPrice:
+    case LINE_FIELDS_MAP.listUnitPriceElectronic:
+    case LINE_FIELDS_MAP.additionalCost:
+    case LINE_FIELDS_MAP.poLineEstimatedPrice:
+      return (
+        <AmountWithCurrencyField
+          currency={currency}
+          amount={value}
+        />
+      );
+    case LINE_FIELDS_MAP.discount:
+      if (!value) return <NoValue />;
+
+      return get(source, 'cost.discountType') === DISCOUNT_TYPE.percentage
+        ? `${value}%`
+        : (
+          <AmountWithCurrencyField
+            currency={currency}
+            amount={value}
+          />
+        );
+    default:
+      if (value === true) {
+        return 'true';
```

Comment:
should it be translated?

Reply:
It's a good question what level of translation is needed. For example, Chinese labels might not help if they are sent to english speaking vendor

Reply:
added translation